<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsNewDatabaseTableNameWidget</class>
 <widget class="QWidget" name="QgsNewDatabaseTableNameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>629</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QToolBar" name="mBrowserToolbar">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
     <addaction name="mActionRefresh"/>
    </widget>
   </item>
   <item>
    <widget class="QgsBrowserTreeView" name="mBrowserTreeView"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>New table name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mNewTableName">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>name of the new table</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mValidationResults">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Validation results</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="mActionRefresh">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="mActionShowFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionFilter2.svg</normaloff>:/images/themes/default/mActionFilter2.svg</iconset>
   </property>
   <property name="text">
    <string>Filter Browser</string>
   </property>
   <property name="iconText">
    <string>Filter Browser</string>
   </property>
   <property name="toolTip">
    <string>Filter Browser</string>
   </property>
  </action>
  <action name="mActionCollapse">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionCollapseTree.svg</normaloff>:/images/themes/default/mActionCollapseTree.svg</iconset>
   </property>
   <property name="text">
    <string>Collapse All</string>
   </property>
   <property name="toolTip">
    <string>Collapse All</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsBrowserTreeView</class>
   <extends>QTreeView</extends>
   <header>qgsbrowsertreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
